openreview/conference/builder.py

```diff
@@ -1212,6 +1212,14 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
         invitation = self.invitation_builder.set_reviewer_recruiter_invitation(self, options)
         invitation = self.webfield_builder.set_recruit_page(self.id, invitation, self.get_homepage_options(), options['reviewers_name'])
 
+        contact_email = 'info@openreview.net'
+        try:
+            request_form = self.client.get_note(self.request_form_id)
+            if 'contact_email' in request_form.content.keys():
+                contact_email = request_form.content['contact_email']
```

Comment:
here, you can se the get function that doesn't throw a KeyError when the key doesn't exist in the dictionary:

`request_form.content.get('contact_email', None)`

Default value is always None but you can use other values, for example: 'info@openreview.net' but I would prefer to throw an error.

Reply:
As in - throw an error when there is no `contact_info` field in the note?

Reply:
Just put the contact_info as a required parameter of recruit_reviewers.